package com.example.examenprogramovil.navigation

sealed class Screen(val route: String) {
    object LogInPage : Screen("login")
    object GithubScreen : Screen("github")
    object MoviesScreen : Screen("movies")
    object Dollar: Screen("dollar")
}